src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,42 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    this.hidden = false;
```

Comment:
Only Herbivores have `hidden` flag.
Right now your Carnivores will be have it too.

`Create a Herbivore class. Herbivore has a method of hide, which changes the hidden property of the beast and helps to hide from carnivores.`